TERMS.md

```diff
@@ -0,0 +1,52 @@
+As a work of the United States Government, this package (excluding any
+exceptions listed below) is in the public domain within the United States.
+Additionally, we waive copyright and related rights in the work worldwide
+through the [CC0 1.0 Universal public domain dedication][CC0].
+
+Software source code previously released under an open source license and then
+modified by CFPB staff or its contractors is considered a "joint work"
+(see 17 USC § 101); it is partially copyrighted, partially public domain,
+and as a whole is protected by the copyrights of the non-government authors and
+must be released according to the terms of the original open-source license.
+Segments written by CFPB staff, and by contractors who are developing software
+on behalf of CFPB are also in the public domain, and copyright and related
+rights for that work are waived through the CC0 1.0 Universal dedication.
+
+For further details, please see the CFPB [Source Code Policy][policy].
+
+
+## CC0 1.0 Universal Summary
+
+This is a human-readable summary of the [Legal Code (read the full text)][CC0].
+
+### No Copyright
+
+The person who associated a work with this deed has dedicated the work to
+the public domain by waiving all of his or her rights to the work worldwide
+under copyright law, including all related and neighboring rights, to the
+extent allowed by law.
+
+You can copy, modify, distribute and perform the work, even for commercial 
+purposes, all without asking permission. See Other Information below.
+
+### Other Information
+
+In no way are the patent or trademark rights of any person affected by CC0,
+nor are the rights that other persons may have in the work or in how the
+work is used, such as publicity or privacy rights.
+
+Unless expressly stated otherwise, the person who associated a work with
+this deed makes no warranties about the work, and disclaims liability for
+all uses of the work, to the fullest extent permitted by applicable law.
+When using or citing the work, you should not imply endorsement by the
+author or the affirmer.
+
+[policy]: https://github.com/cfpb/source-code-policy/
+[CC0]: http://creativecommons.org/publicdomain/zero/1.0/legalcode
+
+
+## Exceptions
+
+_Source code or other assets that are excluded from the TERMS should be listed
+here. These may include dependencies that may be licensed differently or are
+not in the public domain._
```

Comment:
Cool, most scripts in `/scripts` are by the Hubot team. I deleted all third-party ones and added their names to `hubot-scripts.json` so that they'll be automatically pulled from the hubot registry (dunno why they weren't there to begin with). There was one not in the hubot scripts registry that I added to `TERMS.md`.